azext_edge/edge/params.py

```diff
@@ -530,6 +530,11 @@ def load_iotops_arguments(self, _):
             arg_group="TLS",
             type=int,
         )
+        context.argument(
+            "template_path",
+            options_list=["--template-file"],
+            help="The path to a custom IoT Operations deployment template. Intended for advanced use cases.",
```

Comment:
no parameter file option?

Reply:
Didn't want to include a separate parameter from file option because CLI parameters are what should be used. Also, you can bake custom parameter values in the template rather than a separate param file.

Reply:
Wanted to make sure cause there may be demand (especially with how portal defaults to generating a template and param file when you export templates)
![image](https://github.com/Azure/azure-iot-ops-cli-extension/assets/73560279/97f770f1-0099-4fbb-acb4-3c8079309f6e)

Reply:
Also please include a help example demonstrating at least the template param with the other CLI inputs (you can add a note that if the file already has the params, then you can omit them)

Reply:
My concern with adding an example, is we don't want to promote `--template-file` because in most cases a user should deploy with the built-in template. `--template-file` is intended for advanced use cases primarily internally.